config.mk

```diff
@@ -6,6 +6,9 @@ MANPREFIX = ${PREFIX}/share/man
 # leave empty to install into your home folder
 TERMINFO := ${DESTDIR}${PREFIX}/share/terminfo
 
+# the termcap file is shared among all programs
+TERMCAP = /usr/share/misc/termcap
```

Comment:
This is bad. It breaks staged installs. You need to install it under ${DESTDIR}, and, I think, conditionally.

Something in lines "WANT_TERMPCAP ?= no" in config.mk